processor/blobuploadprocessor/metadata.yaml

```
@@ -0,0 +1,21 @@
+type: blobuploadconnector
+scope_name: otelcol/blobuploadconnector
+
+status:
+  class: processor
+  stability:
+    # Implementation still in-progress. Please follow along at:
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33737
+    development: [traces, logs]
+  distributions: [contrib]
```

adding this to contrib is done after the implementation is complete (it reaches alpha).
```suggestion
  distributions: []
```